### PR TITLE
multi-boot: update to Ubuntu 18.04, Mint 19.1, and Win Server 2019; validate use cases.

### PR DESCRIPTION
**Pull request by mvincerx**

### CODE REVIEW

**Comment by mvincerx** [comment]
@linjiach, Will you please contribute to this PR, like we discussed? I added a placeholder document for `multi-boot-fed`, which you can draft the multi-boot use case for Fedora Workstation.
https://getfedora.org/en/workstation/.  If questions, just let me know. 

I'm still editing `multi-boot` landing page, and I will add new content for `multi-boot-win`.

File: source/clear-linux/tutorials/multi-boot/multi-boot-fed.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. _multi-boot-fed:
+
+Install Fedora\* Workstation
```
Comment (krisdale) [inline source/clear-linux/tutorials/multi-boot/multi-boot-fed.rst]: Sentence case for titles.

File: source/clear-linux/tutorials/multi-boot/multi-boot.rst [hunk 2]
```diff
@@ -3,19 +3,29 @@
 Multi-boot |CL-ATTR| with other operating systems
```
Comment (krisdale) [inline source/clear-linux/tutorials/multi-boot/multi-boot.rst]: Update title to not use Clear Linux OS e.g. "Multi-boot with other operating systems"

**Comment by tpleavitt** [comment]
Please add a warning that we will destroy the contents of your hard drive (all partitions)

**Comment by mvincerx** [comment]
@bktan8, @odwdinc, please review at your earliest convenience.

**Comment by mvincerx** [comment]
@bktan8. Will you have time to review this before Wed. meeting?

**Comment by bktan8** [comment]
@mvincerx - On the surface it looks fine, but I think we need to have someone test the instructions.  Let me see if I can find someone.  Can you render the document in a PDF?

**Comment by mvincerx** [comment]
See my email. If you find someone, please refer that person to make comments directly on this PR. Thanks

**Comment by mvincerx** [comment]
@odwdinc Do you have the bandwidth to test 3 use cases identified here?

File: source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst [hunk 3]
```diff
@@ -56,87 +56,84 @@ tutorial.
...
+#. Add the following lines to the :file:`ubuntu.conf` file:
 
-         sudoedit /boot/efi/loader/entries/ubuntu.conf
+   .. code-block:: bash
 
-      Add the following lines to the :file:`ubuntu.conf` file:
+      title Ubuntu 18.04 LTS Desktop
 
-      .. code-block:: console
+      efi /EFI/ubuntu/grubx64.efi
```
Comment (mvincerx) [inline source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst]: @bwarden, @evil-brain, Can either of you see why the above command won't work--or requires an additional step--after installing more OSes after Clear Linux OS? This Ubuntu multi-boot was confirmed to work prior to release of new `live-desktop` and `live-server` isos; however, it's not working now. 
Full render, see: https://github.com/mvincerx/clear-linux-documentation/blob/8019b1a16f74c2cad092ab85bd11fdea0a89a628/source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst

**Comment by mvincerx** [comment]
@mvincerx and @bktan8 will re-open this PR, after further testing, per discussion on 05-31-19.

**Pull request closed without merging**